typings/index.d.ts

```diff
@@ -1006,6 +1006,7 @@ declare module 'discord.js' {
 		public size: number;
 		public url: string;
 		public width: number | null;
+		public spoiler: boolean;
```

Comment:
Now since it's a getter:

```suggestion
		public readonly spoiler: boolean;
```